File: PHP/Exercicio002.php
<!DOCTYPE html>
<html>

<body>

  <form method="post">
    Entre com o primeiro Inteiro:</br>
    <input type="text" id="NUM1" name="NUM1" required></br>
    Entre com o segundo Inteiro:</br>
    <input type="text" id="NUM2" name="NUM2" required></br>   
    <button>Trocar</button></br>
  </form>
<?php

  $NUM1 = $_POST['NUM1'];
  $NUM2 = $_POST['NUM2'];
  
  $AUX = $NUM1;
  $NUM1 = $NUM2;
  $NUM2 = $AUX;

  echo "O valor de NUM1 = " . $NUM1 . "</br>";
  echo "O valor de NUM2 = " . $NUM2 . "</br>";
?>
</body>
</html>

File: JAVA/Exercicio002.java
package main;

import java.util.Scanner;

public class Exercicio002 {
	
	public static void main(String args[]) {
	
		int NUM1, NUM2, AUX;
		
		try(Scanner teclado = new Scanner(System.in)){
			System.out.println("Entre como primeiro n.");
			NUM1 = teclado.nextInt();
			System.out.println("Entre como segundo n.");
			NUM2 = teclado.nextInt();
			teclado.close();
		}
		AUX = NUM1;
		NUM1 = NUM2;
		NUM2 = AUX;
		System.out.println("Valor de NUM1 = " + NUM1);
		System.out.println("Valor de NUM2 = " + NUM2);
	
	}
}
